mitosheet/src/components/taskpanes/Merge/MergeTaskpane.tsx

```diff
@@ -102,20 +109,30 @@ export const getDefaultMergeParams = (sheetDataArray: SheetData[], _sheetIndexOn
         merge_key_column_ids: suggestedMergeKeys ? [suggestedMergeKeys] : [],
         selected_column_ids_one: selectedColumnIDsOne,
         selected_column_ids_two: selectedColumnIDsTwo,
+        destination_sheet_index: undefined
     }
 }
 
 
 const MergeTaskpane = (props: MergeTaskpaneProps): JSX.Element => {
 
+    console.log('props: ', props.existingMergeParams)
+
     const {params, setParams, error} = useLiveUpdatingParams<MergeParams, MergeParams>(
-        () => getDefaultMergeParams(props.sheetDataArray, props.selectedSheetIndex),
+        () => props.existingMergeParams === undefined ? getDefaultMergeParams(props.sheetDataArray, props.selectedSheetIndex) : props.existingMergeParams,
```

Comment:
Instead of this, let's do it like Pivot, where we pass the existingPivotParams to the getDefaultParams function. And handle this case inside that function.